webrtc.html

```diff
@@ -1390,26 +1435,16 @@
           <dd>
             <p>Adds a new stream to the RTCPeerConnection.</p>
 
-            <p>When the <dfn id="dom-peerconnection-addstream"><code title=
-            "">addStream()</code></dfn> method is invoked, the user agent MUST
-            run the following steps:</p>
+            <p>When
+            the <dfn id="dom-peerconnection-addstream"><code>addStream()</code></dfn>
+            method is invoked, the user agent MUST
+            <a>queue an RTCPeerConnection task</a> to run the following steps:</p>
```

Comment:
This needs a bit more work. I don't think we should do that before addStream is replaced by addTrack().